### PR TITLE
Fetch code/data at runtime

### PR DESCRIPTION
**Pull request by blairdrummond**

### CODE REVIEW

File: get-covid-data/Dockerfile [hunk 1]
```diff
@@ -30,7 +30,7 @@ RUN wget https://dl.min.io/client/mc/release/linux-amd64/mc && \
     mv mc /usr/bin/
 
 # COPY data    /data
-COPY src     /src
+# COPY src     /src
```
Comment (brendangadd) [inline get-covid-data/Dockerfile:33]: Remove lines in favour of commenting them out.

File: get-covid-data/main.sh [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
-#     out.json has as "result" field, which is 0 or 1, depending
-#     on whether the random point is inside or outside the circle.
-#
-# Blair Drummond
-#
+
+# Get the source
+git clone --depth=1 \
+    https://github.com/kennethchu-statcan/covid19/ \
+    /tmp/covid19 || exit 1
```
Comment (brendangadd) [inline get-covid-data/main.sh:11]: Might be worth either tossing an error message on failure or choosing an error code that you will recognize down the road as this type of failure. I could see someone pulling out there hair wondering what's wrong with their code when it was just a network error. 😄

File: get-covid-data/Dockerfile (hunk 1 above)
Comment (blairdrummond) [reply get-covid-data/Dockerfile:34]: Leaving them in with an added comment to suggest that that's where a Dockerfile might get modified in another usecase

File: get-covid-data/main.sh (hunk 2 above)
Comment (blairdrummond) [reply get-covid-data/main.sh]: Yes, especially with Istio that's a good idea :-) Point taken haha

**Pull request merged**